source/core/src/main/com/csse3200/game/components/player/PlayerActions.java

```diff
@@ -34,13 +34,7 @@ public void create() {
         entity.getEvents().addListener("walkStop", this::stopWalking);
         entity.getEvents().addListener("attackMelee", this::attack);
         entity.getEvents().addListener("shoot", this::shoot);
-        entity.getEvents().addListener("use1", () -> use(new MedKit()));
```

Comment:
How does one button use all the items?

Reply:
the use method takes an integer and uses the inventory stored at that index in player's usable inventory array.